### PR TITLE
[DOCS-244] Platform Deep Dive book with the Reports section

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@mjang I'm setting up this draft PR as a test for our new Information Architecture. I'll need a link to the Reports guide soon (we discussed this in Slack), so I decided to go ahead and rearrange content based on the new Information Architecture.

I verified that the `cobalt-app-web` repo doesn't contain the links that I'm changing.

### CODE REVIEW

**Comment by mjang-cobalt** [comment]
Suggestion, set this up like we did with https://github.com/cobalthq/cobalt-product-public-docs/blob/main/content/en/BestPractices/template.md

with this in the header

```
toc_hide: true
```

We'll be able to test it with URLs, such as https://<netlify_build_url>/platform-deep-dive/pentests/reports/. Once merged, we can then move / add content, and then remove the `toc_true` property.

**Comment by ana-dashuk-cobalt** [comment]
> We'll be able to test it with URLs, such as https://<netlify_build_url>/platform-deep-dive/pentests/reports/. Once merged, we can then move / add content, and then remove the `toc_true` property.

I've tested all links manually, they work, but it's a good idea.

I want to change this link `https://developer.cobalt.io/getting-started/assets/coverage/#pentest-reports` in the UI earlier, that's why I want to rearrange content.

Do you think we can go ahead and publish it? Then we'll add more content to each section.

**Comment by mjang-cobalt** [comment]
> Do you think we can go ahead and publish it? Then we'll add more content to each section.

I _think_ so. Let me know when this PR is ready for review. Besides a link check I especially want to make sure that changes to the "flow" still work. ("Flow" = what someone needs to do when they set up a pentest.) 

I see you're starting to include `toc_hide`, which should help us set this up in stages

**Comment by ana-dashuk-cobalt** [comment]
> I _think_ so. Let me know when this PR is ready for review. 

The PR is ready for review.

File: content/en/Getting started/glossary.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ In the context of a Cobalt pentest, you can specify one of three options for an
 A potential security flaw in an app or physical hardware. We include findings in vulnerability
 reports, as something that a threat actor can exploit.
 
-When you select [Full Report + Finding Details](/getting-started/what-to-expect/report-contents/#appendix-a---finding-details),
+When you select [Full Report + Finding Details](../../platform-deep-dive/pentests/reports/report-contents/#appendix-a---finding-details),
```
Comment (mjang-cobalt) [inline content/en/Getting started/glossary.md:193]: IMO, absolute links are easier to maintain

```suggestion
When you select [Full Report + Finding Details](/platform-deep-dive/pentests/reports/report-contents/#appendix-a---finding-details),
```

File: content/en/Platform Deep Dive/Pentests/Reports/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+weight: 10
+description: >
+  Get a summary of vulnarabilities in your software.
+---
+
+{{% pageinfo %}}
+Download a pentest report to view security issues that our pentesters found.
+{{% /pageinfo %}}
+
+If you want a pentest report, you generally must set up a test of at least eight (8) credits. If you’ve set up a pentest with fewer credits, you’ll still have access to the non-report items listed in Pentest Expectations.
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/Reports/_index.md:13]: We have a shortcode with numbers of credits in `/layouts/shortcodes/min-credits-report.md`. Maybe:

```suggestion
{{% min-credits-report %}}. If you’ve set up a pentest with fewer credits, you’ll still have access to the non-report items listed in Pentest Expectations.
```

File: content/en/Platform Deep Dive/Pentests/Reports/_index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+We do not create multiple pentest reports for large [assets](../../../getting-started/assets/). For example, if you want separate pentest reports for different APIs, set up different pentests for each API.
+
+## Pentest Report Types
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/Reports/_index.md:17]: Praise: good decision to move this content.

File: content/en/Platform Deep Dive/Pentests/_index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+weight: 20
+description: >
+  Pentests are the cornerstone of our PtaaS platform.
+---
+
+{{% pageinfo %}}
+Launch a pentest to see the benefits of our offering in action.
+{{% /pageinfo %}}
+
+Refer to the [Getting Started](../../getting-started/) guide to set up a pentest in four steps:
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/_index.md:13]: I much prefer stages and not steps. To me, every "click" and data entry is a "step". While we're reducing the number of steps, we still have a few.

```suggestion
Refer to the [Getting Started](../../getting-started/) guide to set up a pentest in four stages:
```

File: content/en/Platform Deep Dive/_index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+title: "Platform Deep Dive"
+linkTitle: "Platform Deep Dive"
+weight: 10
+toc_hide: true
+description: >
+  Explore the features of the Cobalt platform.
+---
+
+{{% pageinfo %}}
+We're working to expand this section of our Product Documentation. Stay tuned.
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/_index.md:11]: Normally I dislike this kind of "We're doing something in the future" kind of statement. 

But we are -- and we have to, in order to complete the process.

**Comment by mjang-cobalt** [comment]
You've done well with this, @ana-dashuk-cobalt . I've included some minor comments (including some things I probably should have fixed before). 

cc @stef-cobalt

**Comment by ana-dashuk-cobalt** [comment]
Thank you for reviewing @mjang . I've also updated some link paths to make them absolute.

**Review by mjang-cobalt — APPROVED** [review]
Thanks!

**Pull request merged**